src/main/java/org/jabref/gui/BasePanel.java

```diff
@@ -1085,11 +1081,6 @@ public void cut() {
         mainTable.cut();
     }
 
-    @Subscribe
-    public void listen(EntryChangedEvent entryChangedEvent) {
-        this.markBaseChanged();
```

Comment:
Interesting that this is not needed anymore. - I tested it locally and the star works fine.

Reply:
EntryChangedEvents are included in the DatabaseChangedEvents (which also trigger a `markBaseChanged`)